### DISCUSSION TITLE
求一个游戏，叫心灵终结，是红警2 尤里复仇的第三方Mod，已经超越了原版

### DISCUSSION THREAD

**Discussion by dxm6006** [Gamuxorg/bbs#205] [投递和请求游戏]
游戏名称：心灵终结 3.3.6
游戏名称(英文)：Mental Omega
下载地址 :（如果有的话）：
国外官网：http://www.mentalomega.com/   
国内汉化整合网站：http://ucloud.marsm.group/
baidu云盘：https://pan.baidu.com/s/1l06uAQCtHCZbDce1ptcJrw?pwd=v084 提取码:v084 
国外官网只提供了Mod补丁，建议到国内汉化网站或者百度云盘上下载整合包。
我现在用的是百度盘上的版本，之前忘记在哪里下载的了，但不知道为什么和汉化网站提供的包容量相差比较大。

**Comment by YMFX3** [comment]
同求

**Reply by chanceyans** [reply]
同求

**Reply by ghost** [reply]
wine直接挂上就是 开opengl的启动器 然后战役有点问题 删除汉化进战役然后留存档 在加汉化进存档就解决了 听别人说好像用wine-tkg更好
